### PR TITLE
feat: delete unused code about showtp #1823

### LINKED ISSUE #1823: Delete unused code about `ShowTp`
the following code should be clean:
- delete `ShowTp` function from src/client/tablet_client.cc and src/client/tablet_client.h
- delete `enable_show_tp` from src/flags.cc and src/client/tablet_client.cc
- delete `percentile_`  from src/client/tablet_client.h

### PR DESCRIPTION
**Pull request by frazie**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

delete unused code about ShowTp, close #1823

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

### CODE REVIEW

**Comment by frazie** [comment]
Hello, I would need clarification on the percentile_ deletion. Kindly review and assist if I might have done it well
Thanks.

File: src/client/tablet_client.h [hunk 1]
```diff
@@ -298,7 +296,7 @@ class TabletClient : public Client {
 
  private:
     ::openmldb::RpcClient<::openmldb::api::TabletServer_Stub> client_;
-    std::vector<uint64_t> percentile_;
+    std::vector<uint64_t>;
```
Comment (aceforeverd) [inline src/client/tablet_client.h]: wrong statement

Comment (frazie) [reply src/client/tablet_client.h]: so should I delete line 301 only? @aceforeverd

**Review by vagetablechicken — CHANGES_REQUESTED** [review]
https://github.com/4paradigm/OpenMLDB/runs/6526550407?check_suite_focus=true

some errors in building, you should fix it.

File: src/client/tablet_client.h (hunk 1 above)
Comment (dl239) [reply src/client/tablet_client.h]: hi @frazie, you should delete this line

**Review by vagetablechicken — APPROVED** [review]

**Review by dl239 — APPROVED** [review]

**Pull request merged**